src/components/views/rooms/EventTile.js

```diff
@@ -152,6 +160,32 @@ module.exports = React.createClass({
         return false;
     },
 
+    componentWillUnmount: function() {
+        dispatcher.unregister(this.dispatcherRef);
+    },
+
+    onAction: function(payload) {
+        switch (payload.action) {
+            case 'device_verified':
```

Comment:
Is the ultimate plan to store device verifications on the home server & sync them between devices? If so, presumably this should be an event that comes out of the client so a client can update if you verify someone's device from another of your devices.